.github/CODEOWNERS

```diff
@@ -548,6 +548,12 @@
 /core/templates/services/services.constants.ajs.ts @vojtechjelinek
 /core/templates/services/services.constants.ts @vojtechjelinek
 
+
+# Dynamic Feature Gating project
+/core/storage/platform_parameter/ @MegrezZhu
```

Comment:
Oppiabot is already adding a comment: https://github.com/oppia/oppia/pull/9876#issuecomment-661125930.

Reply:
Though @jameesjohn would it be possible to link to specific lines where the model is added rather than a common link?

Reply:
> Oppiabot is already adding a comment: #9876 (comment).

I think @seanlip meant commenting about the code owner addition.


> Though @jameesjohn would it be possible to link to specific lines where the model is added rather than a common link?


I don't think we can get the exact line because the file diff doesn't contain them.

Reply:
Yes that is correct -- I did mean the code owner addition.

@jameesjohn @ankita240796 What are your future plans regarding this? Is there a tracking issue?